packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -807,6 +807,36 @@ export default class Meetings extends WebexPlugin {
     );
   }
 
+  /**
+   * Creates a noise reduction effect
+   *
+   * @param {Object} options optional custom effect options
+   * @returns {Promise<effect>} noise reduction effect.
+   * @public
+   * @memberof Meetings
+   */
+  createNoiseReductionEffect = async (options = {}) => {
```

Comment:
wouldn't it be better for these "create effects" functions be all in a new separate ts file? We currently have createMicrophoneTrack, createCameraTrack, createDisplayTrack in media-helpers and they are just re-exported in plugin-meetings. Shouldn't we do the same with effects? They could be in media-helpers, just in a separate file, not webrtc-core.ts

Reply:
Yes this could be an approach we could take but what I discussed with Arun is that we were planning to rename `media-helpers` package to `internal-media-helpers` as the effects imported and exported from media-helpers package won't run without an access_token and we want developers to use it with `meetings` plugin once this PR is merged as we have created a wrapper here which doesn't require token from user.

Also if we move effects wrapper logic there then developers using `meetings` package have to import `media-helpers` package also but with the current approach they can directly access it on `meetings` object.

I'm open for more discussion if we want to move this logic on media-helpers package.

Reply:
developers don't have to import media-helpers, you can export stuff from media-helpers in plugin-meetings - that's what we're already doing for createMicrophoneTrack, createCameraTrack, createDisplayTrack etc

Reply:
`media-helpers` is just plain plugin for exporting tracks and effects related things and we won't have access to `webex` object there. That's why `createEffect` logic is in plugin-meetings where we have access to webex object.

Reply:
ok, fine let's keep in in meetings